index.html

```diff
@@ -1228,6 +1228,34 @@ <h2>A project for first-time contributions</h2>
         </div>                  
         <!-- ________ slunickova's card END ________  -->
 
+        <!-- ________ 5r1n1's card START ________  -->
+        <div class="card">
+
+            <p class="name">5r1n1v@5</p>
+            <p class="contact">
+                <i class="fa fa-twitter"></i>
+                <a href="YOUR twitter account link" target="_blank">@SGK9677118997</a>
```

Comment:
You forgot to add the href of your Twitter account here. 
Everything else is great, when you have it ready I will merge your PR.